CHANGELOG.rst

```diff
@@ -10,6 +10,10 @@ and this project adheres to `Semantic Versioning <https://semver.org/spec/v2.0.0
 ------------
 Changed
 ^^^^^^^
+- Now time will be allow second precision in storage on uint32 from 01/01/1950 to 01/01/2086
```

Comment:
do you mean "time will allow second precision"  or "time precision extended from day to second" ?

Reply:
I am not sure to understand differences between this 2 sentence

Reply:
It's becouse second is also 2nd I think (second decimal 0.01, even decimal is not written). "Time precision extended from day to second" remove the ambiguity